### DISCUSSION TITLE
Issues with liveserver

### DISCUSSION THREAD

**Discussion by vcbobby** [microsoft/WSL#6713] [Q&A]
Hello, i want to report issues with the extension live server.

I open VS Code from my window subsystem for linux, and when i try to open my html file from the extension live server, the extension didn't load in my browser, but if i open the same html file from vs code normally everything is ok. I good like a revew from this case, please

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by yandeu** [comment]
Can you try if it works using the updated [Five Server](https://marketplace.visualstudio.com/items?itemName=yandeu.five-server)?
